### PR TITLE
testing pullrequest

### PR DESCRIPTION
**Pull request by sakthivelmt**

### CODE REVIEW

File: Sample.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class Sample {
   public static void main(String[] args) {
-    System.out.println("Hello");
+    System.out.println("Forked form javaplug");
```
Comment (javaplug) [inline Sample.java]: sample comment

File: Sample.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 class Sample {
```
Comment (javaplug) [inline Sample.java]: testing comments

Comment (sakthivelmt) [reply Sample.java]: done

File: Sample.java (hunk 1 above)
Comment (sakthivelmt) [reply Sample.java]: done

**Pull request merged**